audit/projects/k8s-infra-e2e-scale-5k-project/services/compute/project-info.json

```diff
@@ -1,6 +1,10 @@
 {
   "commonInstanceMetadata": {
     "items": [
+      {
+        "key": "enable-oslogin",
+        "value": "TRUE"
+      },
```

Comment:
This is required by the canary prowjobs for sig-scalability. This ensure the SA `prow-build@k8s-infra-prow-build.iam.gserviceaccount.com` can login in the instances created in the project. I need to add a `ensure_project_oslogin` bash function.

Reply:
Opened followup issue: https://github.com/kubernetes/k8s.io/issues/2278